project/src/main/puzzle/playfield-fx.gd

```diff
@@ -185,6 +201,10 @@ Calculates the new light pattern and refreshes the tile maps.
 func _refresh_tile_maps() -> void:
 	$BgStrobe.color = Utils.to_transparent(_color)
 	
+	if not _color_tile_indexes:
+		push_warning("PlayfieldFx._color_tile_indexes is empty, refreshing.")
```

Comment:
Add a comment; we can probably remove this kludge eventually if this warning never happens.

Reply:
I've removed the check; it seems overly paranoid. If the issue appears to recur, we can add something like this back in.